our-brothers-web/src/app/agenda/page/agenda-page.component.ts

```diff
@@ -1,18 +1,55 @@
-import { Component, OnInit, ChangeDetectionStrategy } from '@angular/core';
+import { Component, OnInit, ChangeDetectionStrategy, ElementRef, AfterViewInit } from '@angular/core';
 import { Quote, QuotesService } from './quotes.service';
+import { timeInterval } from 'rxjs/operators';
+import { interval } from 'rxjs';
 
 @Component({
   selector: 'app-agenda-page',
   templateUrl: './agenda-page.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
   styleUrls: ['./agenda-page.component.scss']
 })
-export class AgendaPageComponent implements OnInit {
+export class AgendaPageComponent implements OnInit, AfterViewInit {
   pagedExcerpts: { page: Quote[] }[];
+  public slideIndex: number;
 
-  constructor(private quotesService: QuotesService) {}
+  constructor(private quotesService: QuotesService, private elementRef: ElementRef) {
+    this.slideIndex = 1;
+  }
 
   ngOnInit() {
     this.pagedExcerpts = this.quotesService.pagedQuotes;
+
+    const seconds = interval(10000);
```

Comment:
I don't see a reason for the declaration of the `seconds` variable. consider to inline it with the next line.